x/staking/types/expected_keepers.go

```diff
@@ -6,18 +6,18 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/staking/exported"
 )
 
-// expected coin keeper
+// expected coin keeper (noalias)
```

Comment:
can we define the noalias at the top of this file instead ?

Reply:
no unfortunately not because we (currently) are unable to have everything in this file without an alias, some other modules depend on knowing the expected keepers in staking